features/paypal/paypal.feature

```diff
@@ -0,0 +1,5 @@
+Feature: Transaction with PayPal
+
+  @javascript
+  Scenario: Test transaction with PayPal
+    Then I expect transaction with PayPal test to pass
```

Comment:
I'd love to get rid of this unnecessary boilerplate since we're not utilizing the Gherkin DSL anymore. That can be done in a later PR though as it requires a bit of thinking how to separate from non-Selenium based tests.